requirements-dev.txt

```diff
@@ -15,3 +15,5 @@ mkdocs-open-in-new-tab==1.0.3
 pynvim==0.5.0
 jedi-language-server==0.41.4
 httpx==0.27.2
+chainlit==1.3.2
```

Comment:
Should move to a `requirements.txt` under `chainlit_app`

Reply:
Yes, it should be here. I moved it because it couldn't reference correctly, and this was done during a debugging session with Mark